### PR TITLE
upgraded docusaurus to 2.2.0

### PR DESCRIPTION
**Pull request by rizbihassan**
## Summary of Changes

Please write a sentence or two summarizing the proposed change.

Addresses [ORCA-596: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/ORCA-596)

## Changes

* Detailed list or prose of changes
* ...

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

confirmed the version using `npx docusaurus --version`
deployed the website and format looks good

## Checklist:

- [x ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [xx ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
    - [ ] Automated tests passing (if not fork)
    - [ ] Manual testing/integration testing passes (if forked)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] My code has passed security scanning
    - [ ] Snyk
    - [ ] git-secrets

### CODE REVIEW

**Comment by rizbihassan** [comment]
Compiled with problems:
X

ERROR in ./node_modules/@docusaurus/theme-classic/lib/theme/MDXComponents/index.js 6:444-480

Module not found: Error: Can't resolve '@theme/Mermaid' in '/Users/rhassan/cumulus-orca/website/node_modules/@docusaurus/theme-classic/lib/theme/MDXComponents'

**Review by AndrewDorn — APPROVED** [review]

File: website/package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.1.0",
-    "@docusaurus/preset-classic": "2.1.0",
+    "@docusaurus/core": "2.2.0",
+    "@docusaurus/preset-classic": "2.2.0",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.1.1",
     "prism-react-renderer": "^1.3.3",
```
Comment (bhazuka) [inline website/package.json:21]: Based on the current version of Docusaurus 2.2.0 can the following libraries be updated?
-  "@mdx-js/react": "^1.6.22",
-  "clsx": "^1.1.1",
-  "prism-react-renderer": "^1.3.3",
-  "react": "^17.0.2",
-  "react-dom": "^17.0.2",
-  "react-zoom-pan-pinch": "^2.1.3"



Also you will need to run `npm install` and add the newest/updated [package-lock.json](https://github.com/nasa/cumulus-orca/blob/master/website/package-lock.json) file to this PR.

**Review by bhazuka — CHANGES_REQUESTED** [review]
Looks good so far. May be able to upgrade some of our add-ons but will need to verify that v2.2.0 can handle the version. 

Also, you will need to add the `package-lock.json` file with the updated libraries.

File: website/package.json (hunk 1 above)
Comment (rizbihassan) [reply website/package.json:21]: - "@mdx-js/react": "^1.6.22", -> 2.1.1
ERROR:
`This page crashed.
(0 , _mdx_js_react__WEBPACK_IMPORTED_MODULE_2__.mdx) is not a function`

- "clsx": "^1.1.1" -> 1.2.1
updated

- "prism-react-renderer": "^1.3.3" -> 1.3.5
updated

- "react": "^17.0.2" -> 18.2.0
`React 16.8.4
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.4 || ^17.0.0" from @docusaurus/core@2.2.0`

- "react-dom": "^17.0.2" -> 18.2.0
`npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.2.0" from react-dom@18.2.0
npm ERR! node_modules/react-dom
npm ERR!   react-dom@"^18.2.0" from the root project`

- "react-zoom-pan-pinch": "^2.1.3" 
already has the latest version

**Review by bhazuka — APPROVED** [review]

**Pull request merged**